.github/workflows/action.yml

```diff
@@ -0,0 +1,18 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Run the Maven verify phase
+        run: mvn --batch-mode --update-snapshots verify
```

Comment:
mi duda es si no necesitas tb correr mvn package
que hace el --update-snapshots

Reply:
**La documentación dice:**

`The starter workflow will run the package target by default. In the default Maven configuration, this command will download dependencies, build classes, run tests, and package classes into their distributable format, for example, a JAR file.`

**Respecto al "verify":**
`If you use different commands to build your project, or you want to use a different target, you can specify those. For example, you may want to run the verify target that's configured in a pom-ci.xml file.`

Luego buscando especificamente lo que preguntaste encontré en internet:

**Snapshot:**
`A Maven snapshot is a special version of a Maven package that refers to the latest production branch code. It is a development version that precedes the final release version.`

**Respecto al comando completo --update-snapshots:**
`Forces a check for updated releases and snapshots on remote repositories`

Reply:
ah, ok, ya lo hace solo el package... ok
Excelente